### DISCUSSION TITLE
https://budibase.github.io/budibase/ throw 404

### DISCUSSION THREAD

**Discussion by datacoderid** [Budibase/budibase#3794] [Help]
Hi, I try to install budibase in kubernetes but always got installation failed.
When I access the repo (https://budibase.github.io/budibase/) it give me 404 page not found.
Anyone has the updated link?

**Comment by shogunpurple** [comment]
Hi @datacoderid - seems to be an issue with the helm repo. We will publish the latest version of the budibase helm chart today.

EDIT: This has now been updated - try adding the budibase repo now!
